contracts/TrueReward/AssuredFinancialOpportunity.sol

```diff
@@ -85,7 +85,7 @@ contract AssuredFinancialOpportunity is FinancialOpportunity, AssuredFinancialOp
     }
 
     function getBalance() external view returns (uint) {
-        return _getBalance();
+        return _getBalance().mul(_perTokenValue()).div(10**18);
```

Comment:
I would move that calculation to `_getBalance()`. In all other places where `_getBalance()` is used, the same calculation is applied.